src/maintainer/knowledge_base.rst

```diff
@@ -1518,3 +1518,11 @@ The tl;dr here is that conda sorts as follows:
 So make sure that you **tag** your package in such a way that the package name
 that conda-build spits out will sort the package uploaded with an ``rc`` label
 higher than the package uploaded with the ``dev`` label.
+
+.. _using_arch_rebuild:
+
+Using ``arch_rebuild.txt``
+==========================
+
+You can add a feedstock to ``arch-rebuild.txt`` if it requires rebuilding with different architectures/platforms (such as ppc64le or aarch64).
```

Comment:
I would clarify that you need to open a PR on https://github.com/conda-forge/conda-forge-pinning-feedstock to propose changes to that file.